### ISSUE TITLE
WindowsOS - User must hit back button or desktop button to change views

### ISSUE THREAD

**Issue by phillsatellite** [Satellite-im/Uplink#263]
**Issue:** If you switch to mobile view and then back to the desktop the UI will not change from mobile until you hit either the back arrow that shouldnt be present still or the desktop button a second time

**How to replicate:**
-Switch to mobile view
-Switch to desktop view
-Hit the back arrow or hit the desktop button a second time 

**Screen recording:**

https://user-images.githubusercontent.com/93608357/217091285-f326c8f0-e687-4bd5-ace4-3fcf2f099a72.mp4

**Comment by sdwoodbury** [comment]
switching to mobile view hides the sidebar. we could make switching to desktop view unhide the sidebar. 

be warned. if we do this. then  if you hide the sidebar in desktop view, switch to mobile view, and then switch back, the sidebar would not be hidden.

**Comment by sdwoodbury** [comment]
on further investigation, switching to desktop already expanded the sidebar. i believe this is a duplicate of #265 
edit: i might be wrong about this. suggest re-evaluating after #265 is fixed.

**Comment by phillsatellite** [comment]
I talked with Stu he helped me better understand the issue going on lol We think it might be good to re-evaluate this bug after https://github.com/Satellite-im/Uplink/issues/265 is fixed since the only place the sidebar is automatically toggled is in the resize event. combining that with the knowledge that the button to switch to desktop expands the sidebar leads me to believe that the resize event is causing both bugs -notes from Stu! 🚀

**Comment by phillsatellite** [comment]
Going to close for now seems to be duplicate of #265 when testing its the same behavior as 265 and same steps to temp fix. Can always re open if we need to.
